src/d3d9/d3d9_device.cpp

```diff
@@ -2615,7 +2615,22 @@ namespace dxvk {
     if (unlikely(!PrimitiveCount))
       return S_OK;
 
-    PrepareDraw(PrimitiveType);
+    bool dynamicSysmemVBOs = true;
```

Comment:
If every vbo is nullptr, this should be false to handle the case of Draws with 0 vertex buffers bound, right?